### DISCUSSION TITLE
content/cherokee-tts-update--2--ims-toucan-/

### DISCUSSION THREAD

**Discussion by giscus** [CherokeeLanguage/CherokeeLessons.com#7] [General]
# content/cherokee-tts-update--2--ims-toucan-/

http://127.0.0.1:1111/content/cherokee-tts-update--2--ims-toucan-/

**Comment by michael-conrad** [comment]
What do y'all think of the audio?

**Reply by TroyFallin** [reply]
You're a genius!  ꮹꮩ ꭰꮄ ꮩꮎꮣꭺꮂꭲ

**Reply by TroyFallin** [reply]
By the way, I was taught by a Cherokee elder whose first language was Cherokee and he had been a soldier in WW2, a code talker. His name was Alex Coffee and he is the reason that I have a good ear for how the language should sound. I'm far from fluent though. I am amazed at how well these computer generated words sound.

**Comment by michael-conrad** [comment]
I'm glad to here it is performing well according to someone with a practiced ear for the language.

The biggest challenge right now is a general lack of properly transcribed audio data. Where the audio data is transcribed using an orthography that reflects actual pronunciation including tones and cadence. The Modified Community Orthography is one such orthography.

**Reply by michael-conrad** [reply]
Other audios exist, but is not accompanied with a pronunciation transcription.
